### PR TITLE
Markdown & MarkdownEditor to less. Remove styled-components SSR/dependencies.

### LINKED ISSUE #41: Replace Styled Components with Less
We're currently implementing styles in a few different ways, including less, styled components, and inline styles. This should be consolidated to a single form, less.

- [ ] Investigate tslint flag for preventing inline styles
- [ ] Replace styled components in favor of basic less styles

### PR DESCRIPTION
**Pull request by AMStrix**
Closes #41.

### What this does
- convert `Markdown` & `MarkdownEditor` styles to less
- remove styled-components SSR code
- remove styled-components & related dependencies

### Testing
- update deps, `yarn`
- make sure `yarn dev` and `yarn build` run clean
- make sure create>details (preview) looks correct
screen shot 2018-09-22 at 1 11 44 pm
- make sure create>review details render correctly
screen shot 2018-09-22 at 1 13 27 pm
- make sure proposal>comments (preview) looks correct
screen shot 2018-09-22 at 1 16 49 pm

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -125,7 +124,6 @@
     "showdown": "^1.8.6",
     "stats-webpack-plugin": "^0.7.0",
     "style-loader": "^0.23.0",
-    "styled-components": "^3.4.6",
```
Comment (dternyak) [inline frontend/package.json]: 😱 awesome!

Comment (AMStrix) [reply frontend/package.json]: 😆

**Comment by dternyak** [comment]
I don't think this was introduced by this PR, but I noticed a regression on the "Show More" section of a proposal description.

Show me will be hidden initially when navigating to it, but not when loading the page directly.



GIF to show steps to reproduce.

**Comment by AMStrix** [comment]
I'll squeak in a fix this afternoon.

**Pull request merged**